R/TealAppDriver.R

```diff
@@ -365,16 +372,30 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
 
       if (identical(slices_suffix, "selection_manual")) {
         checkmate::assert_numeric(input, len = 2)
+        extra_formals <- formals(app$set_inputs)
+
+        checkmate::assert_choice(dots$priority_, formals(self$set_inputs))
+
+        dots <- rlang::list2(...)
         self$run_js(
           sprintf(
-            "Shiny.setInputValue('%s:sw.numericRange', [%f, %f], {priority: 'event'})",
+            "Shiny.setInputValue('%s:sw.numericRange', [%f, %f], {priority: '%s'})",
             slices_input_id,
             input[[1]],
-            input[[2]]
+            input[[2]],
+            priority_ = dplyr::coalesce(dots$priority_, "input", .size = 1)
```

Comment:
should we add `dplyr`in suggest ?

Reply:
Oh shoot! I could swear it was a dependency, thanks for this .

Strange that check is not failing :thinking:

Reply:
We can use `rlang::%||%` instead, I wanted to avoid it as it requires backticks to call directly, and adding `@importFrom rlang %||%` seems overkill for a single use

Reply:
While it might seem unconventional use `rlang::<backtick>%||%<backtick>(dots$priority_, "input")`, I'm open to this approach. However, it's important to note testthat dir,  also makes use of `dplyr::inner_join`. 

Instead of relying on `dplyr::coalesce`, I'd suggest employing either `rlang::<backtick>%||%<backtick>(dots$priority_, "input")` or opting for a straightforward conditional statement: if (is.null(dots$priority_)) "input" else dots$priority_.